src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -8,28 +8,32 @@ const ButtonGroup = styled.div`
   vertical-align: middle;
   isolation: isolate;
 
-  && > * {
+  && > button,
```

Comment:
@broccolinisoup is it possible these could be `a` tags as well? You could add a test story case for that and see.

Reply:
Oo great catch @langermank! Definitely, they can be `a` tags. I'll update the code.